streams/src/test/java/org/apache/kafka/streams/processor/internals/StandbyTaskTest.java

```diff
@@ -554,6 +556,95 @@ public void shouldRestoreToKTable() throws IOException {
         );
     }
 
```

Comment:
The following two tests have rather much code in common. Please try to extract common parts to methods.